src/components/GlobalError/GlobalError.js

```diff
@@ -20,7 +20,7 @@ const GlobalError = ({ meta }) => {
   const { formState } = useFormContext()
   return !isEmpty(formState?.errors) ? (
     <ErrorItem role="alert">
-      {meta.label ||
+      {meta?.label ||
```

Comment:
Why a question mark? The props say it is required. Shouldn't it be required here as well?